src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkedComponents/MeshFilter/MeshFilterBroadcaster.cs

```diff
@@ -27,11 +31,42 @@ protected override void WriteRenderer(BinaryWriter message, byte changeType)
             if (HasFlag(changeType, MeshFilterChangeType.Mesh))

```

Comment:
Have you tested whether this flag gets set when someone modifies the meshFilter.mesh on update/start?

Reply:
No. I have verified that this worked with dynamically-generated meshes in my project, but I didn't take a close look at exactly how the dynamic meshes were created.

I haven't tested changing a mesh after it has already been broadcast and I expect that changes after broadcast would *not* be detected.